tests/BaseTestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Tests;
+
+use PHPUnit\Framework\TestCase;
+
+abstract class BaseTestCase extends TestCase
```

Comment:
I prefered the previous name as it implies it's not a standalone testcase but only the base for testcase